### DISCUSSION TITLE
Increase FastCGI buffer size within NGINX template for Magento 2

### DISCUSSION THREAD

**Discussion by Serializator** [bjarn/jale#58] [Ideas]
**Problem**
In Magento 2 you might get a 502 Bad Gateway with the message "Upstream sent too big header while reading response". Based on from which perspective you look at it you can either see this as a problem in Magento or a lack of buffer size in NGINX.

**Suggestion**
Increase the FastCGI buffer size within NGINX.

```nginx
fastcgi_buffers 16 256k;
fastcgi_buffer_size 256k;
```

**Related**
https://magento.stackexchange.com/questions/268896/upstream-sent-too-big-header-while-reading-response-header-from-upstream

**Comment by bjarn** [comment]
Resolved by #86
